config.lua.dist

```diff
@@ -42,7 +42,7 @@ gameProtocolPort = 7172
 statusProtocolPort = 7171
 maxPlayers = 0
 serverName = "OTServBR-Global"
-motd = "Welcome to the OTServBR-Global!"
+serverMotd = "Welcome to the OTServBR-Global!"
```

Comment:
We have to make sure that this tag is not read elsewhere, like for example in otclient or on websites, if it is used, it is better to keep the old name.